install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot some or all of the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+Re-using the Address::
+
+To re-use the IP address, you must save the IP address of the old docker-registry
+service prior to deleting it, and arrange to replace the newly assigned IP address
+with the saved one in the new docker-registry service.
+
+// NB: Snarfed from <https://github.com/openshift/openshift-docs/issues/1494>.
+. Make a note of the `ClusterIP` for the service.
+
+. Delete the service.
++
+----
+oc delete svc/docker-registry dc/docker-registry
+----
+
+. Create the registry definition in *_registry.yaml_*,
+replacing `<options>` with, for example, those used in step 3 of the
+instructions in the link:#non-production-use[Non-Production use] section.
++
+----
+oadm registry <options> -o yaml > registry.yaml
+----
+
+. Edit *_registry.yaml_*, find the `Service` there,
+and change its `ClusterIP` to the address noted in step 1.
+
+. Create the registry using the modified *_registry.yaml_*.
++
+----
+oc create -f registry.yaml
+----
+
+[[rebooting-the-cluster]]
+Rebooting the Cluster::
+
+If you are unable to re-use the IP address,
+you must reboot some or all of the cluster after creating the new docker-registry service.
+This ensures that the old registry URL, which includes the old IP address,
+is cleared from the cache.
++
+[horizontal]
+Only master:::
+Rebooting only the master avoids unnecessary downtime.
+However, it may leave stale references to the old IP address in
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#master[etcd].
+This is *NORMALLY-HARMLESS* but *MILDLY-UNDESIRABLE-CONSEQUENCE*.
+
+Entire cluster:::
```

Comment:
(from IRC in response to followup question "what is the consequence of stale IP addrs?")

> possibility of failed image pulls (leading to failed container/pod starts) and pushes

and also

> if you restart the master, so it picks up the new ip, new builds & deployments will start using the new ip correctly

Cool, thanks.  Will now fill in the placeholders appropriately.  (The reason for all this fuss is to give the reader a bit of "why" behind the "do this" instructions, part of the effort to preempt some support calls.)